app/src/main/java/com/triviagenai/triviagen/trivia/presentation/results/ResultsScreen.kt

```diff
@@ -43,15 +60,15 @@ fun ResultsScreen() {
                 contentAlignment = Alignment.Center
             ) {
                 CircularProgressIndicator(
-                    progress = score,
+                    progress = score / 25,
                     color = if (score > 0.5f) Color.Green else Color.Red,
```

Comment:
What if we changed these colors to our LightGreen and LightRed so that we have the same colors for right and wrong answers throughout the app?

Reply:
That's a good idea. Those colours were intended for the shadows and are a little brighter. I'll make another version of green and red for for the borders.